UI/EmuScreen.cpp

```diff
@@ -1591,7 +1591,7 @@ bool EmuScreen::hasVisibleUI() {
 	if (g_Config.bEnableCardboardVR || g_Config.bEnableNetworkChat)
 		return true;
 	// Debug UI.
-	if (g_Config.bShowDebugStats || g_Config.bShowDeveloperMenu || g_Config.bShowAudioDebug || g_Config.bShowFrameProfiler || g_Config.bShowControlDebug)
+	if (g_Config.iDebugOverlay != DebugOverlay::OFF)
```

Comment:
This still needs `|| g_Config.bShowDeveloperMenu` right?  Which I would not want to be an alternative to those others.

-[Unknown]